content/docs/06-integrations/00-antrea-cni.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'Antrea CNI'
+metaTitle: 'Antrea CNI'
+metaDescription: 'Antrea CNI network pack for Palette Kubernetes Clusters'
+hiddenFromNav: true
+type: "integration"
+category: ['network']
+logoUrl: 'https://registry.spectrocloud.com/v1/palette-upgrader/blobs/sha256:b6081bca439eeb01a8d43b3cb6895df4c088f80af978856ddc0da568e5c09365?type=image/png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for VMware Kubernetes clusters. Antrea CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility. 
+
+Antrea leverages [Open vSwitch](https://www.openvswitch.org/) to implement pod networking and security features. Open vSwitch enables Antrea to implement Kubernetes network policies efficiently.
+
+<br />
+
+# Supported Versions
+
+**1.9.x**
+
+<br />
+
+# Prerequisites
+
+- Enable the integrated NodeIPAM controller in the Antrea manifest: ``NodeIPAM:enable``.
+- When deploying a cluster using Palette, use the ``podCIDR`` parameter in the Pack section of the Kubernetes manifest. The CIDER IP specified with ``podCIDR`` always takes precedence.
+
+    When deploying a cluster using kubeadm to use Antrea CIDRs, specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the classless inter-domain routing (CIDR). For example: 
+
+    <br />
+
+    ``--pod-network-cidr=10.244.0.0/16``
+
+    <br />
+
+    <WarningBox>
+
+    The CIDER IP specified in Palette with the ``podCIDR`` parameter always takes precedence. 
+    
+    If you want to use Antrea CIDRs, you need to remove values for ``podCIDR`` in the Kubernetes manifest and any specified with ``serviceCIDR`` in the Antrea CNI manifest.  
+    
+    To avoid overlapping your pod network with any of your host networks, you should think of a suitable CIDR block to specify if you deploy a cluster using ``kubeadm init`` with ``--pod-network-cidr <cidr>`` or as a replacement in your network plugin's YAML.
+
+    </WarningBox>
+
+    <br />
+
+- The Open vSwitch kernel module must be present on every Kubernetes node.
+
+
+# Parameters
+
+The Antrea CNI pack supports the following parameters.
+
+| Parameter | Description | Required (Y/N) |
+|-----------|-------------|---------|
+| nodeIPAM:enable | Enables the integrated NodeIPAM controller in the Antrea manifest. The default is `false`. | Y |
+| clusterCIDRs | CIDR ranges for pods in the cluster. The CIDRs can be either IPv4 or IPv6. You can specify up to one CIDR for each IP family. | N |
+| serviceCIDRv6 | IPv6 CIDR ranges reserved for Services. | N |
```

Comment:
@roshanirathi You mentioned ``serviceCIDR`` doesn't work. Should it be removed from the parameters table?